geoips/plugins/modules/coverage_checkers/windbarbs.py

```diff
@@ -59,7 +63,7 @@ def windbarbs(
 
     from geoips.interfaces import interpolators
 
-    interp_plugin = interpolators.get_plugin("pyresample_wrappers.interp_nearest")
+    interp_plugin = interpolators.get_plugin("interp_nearest")
```

Comment:
@jsolbrig Note - hard coded update to an interpolator name here..

Reply:
Got it. Thanks.